hooks/post/10-post_install.rb

```diff
@@ -37,7 +37,7 @@ def new_install?
       say <<MSG
   * To install additional capsule on separate machine continue by running:
 
-      capsule-certs-generate --capsule-fqdn "<%= color('$CAPSULE', :info) %>" --certs-tar "<%= color('~/$CAPSULE-certs.tar', :info) %>"
+      capsule-certs-generate --capsule-fqdn "<%= color('$CAPSULE', :info) %>" --certs-tar "<%= File.join('/root', File.basename('certs.tar.gz')) %>"
```

Comment:
File.basename on a string isn't going to get you anything, do it on `certs_tar` variable which comes from the puppet parameters, you might have to move some things around, since it's defined below this line